urbandict-search/yaml/splitter.py

```diff
@@ -9,4 +9,4 @@
 class Splitter(BaseCrafter):
     def craft(self, text: str, *args, **kwargs) -> Dict:
         word, definitions = text.split('+-=')
-        return dict(text=definitions, meta_info=word.encode())
+        return dict(text=definitions, meta_info=text.encode())
```

Comment:
why this? we are working on an update that will keep this, in the `meta_info` keeping the word

Reply:
I think before we fix the problem, we just use this way to make it work

Reply:
What's your consideration about this meta_info?

Reply:
How is this used in the `app.py`?

Reply:
I'm thinking to do it the same way as southwark. The whole line word+-=definition will be kept in meta_info and be separate again in app.py so word and definition can be fetched

Reply:
Word should be kept as `meta_info` because it is not part of the `embedding`. The separation will be done by finding the actual `word definitions` in the texts (that is what is actually embedded), and the `word` to define is just `meta_info` that is the word refering to `but not encoded`. I want to review southpark example to keep the `character` as `meta_info` so that `Filtering` can be applied to that field.

Reply:
What I'm thinking is that we still use `text=definition` to do the embedding. Because one text is corresponding to one meta_info, we can retrieve both word and definition from meta_info

Reply:
Like put definition both in text as embedding and in meta_info for retrieving

Reply:
Yes, we could, but it is more `educational` to have in meta_info just the word. Why would u duplicate information?

Reply:
besides, it is doing the split twice, is harder to keep track of the changes, if we change the split in one place we need to change in the other.

Reply:
Yes I agree, this is only aimed to avoid the problem and make it work first

Reply:
If it is just as a workaround to have a working solution ASAP I am ok!

Reply:
made the proposed changes in #147 for southpark. Everything works well for southpark except the ugly `chunks[0]`

Reply:
seems urbandict is the only one that cuts text into different chunks

Reply:
> made the proposed changes in #147 for southpark. Everything works well for southpark except the ugly `chunks[0]`

Yes, I think we have the same issue as in `urbandict`, will take a look afterwards